notifier/config.go

```diff
@@ -10,7 +10,7 @@ type Config struct {
 	SendingTimeout   time.Duration
 	ResendingTimeout time.Duration
 	Senders          []map[string]string
-	ImageStore       map[string]string
+	ImageStores      map[string]map[string]string
```

Comment:
Notifier config should not know about the settings of the image storage, he only needs to know what storages are. We use this pattern everywhere. For example, the same notifier does not know anything about RemoteStorage but at the same time, he uses it. Just look at the file `notifier/main.go` and how we use it.

Reply:
For now, we can remove this field because it is no usages